arrays/fizz_buzz_array.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+numbers = (1..100).to_a
+fizz_buzz_nums = []
+
+
+numbers.map do |number|
```

Comment:
So the `map` function applies the loop, and assigns the return result for each number.

    foo = [1, 2, 3]
    bar = foo.map do |number|
      if number.even?
        "Even"
      else
        "Odd"
      end
    end

`bar` is a new array that contains `["Odd", "Even", "Odd"]`.

So line 7 should be `fizz_buzz_numbers = numbers.map do |number|`.  You can then get rid of line #4.